samples/CustomRouteSample.Web/Controllers/CustomRoute_OrdersControlller.cs

```diff
@@ -3,7 +3,7 @@
 
 using Microsoft.AspNet.Mvc;
 
-namespace CustomRouteWebSite.Controllers
+namespace CustomRouteSample.Web.Controllers
```

Comment:
Ok. Let's let the dust settle and then change it up or don't. There's no reason to cling to something that doesn't add value.